services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/ReadBatch.java

```diff
@@ -27,6 +27,14 @@
 import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
 import software.amazon.awssdk.services.dynamodb.model.KeysAndAttributes;
 
+/**
+ * Defines a collection of primary keys for items in a table, stored as {@link KeysAndAttributes}.
+ * <p/>
```

Comment:
I think we should also say how it's used, eg: for BatchGetItems

Reply:
Updated.